Libplanet.Store.Remote/Server/RemoteKeyValueService.cs

```diff
@@ -0,0 +1,84 @@
+using Google.Protobuf;
+using Google.Protobuf.Collections;
+using Grpc.Core;
+using Libplanet.Store.Remote.Extensions;
+using Libplanet.Store.Trie;
+using Microsoft.Extensions.Logging;
+
+namespace Libplanet.Store.Remote.Server
+{
+    public class RemoteKeyValueService : KeyValueStore.KeyValueStoreBase
+    {
+        private readonly ILogger<RemoteKeyValueService> _logger;
+        private readonly IKeyValueStore _baseStore;
+
+        public RemoteKeyValueService(
+            ILogger<RemoteKeyValueService> logger,
+            IKeyValueStore baseStore)
+        {
+            _logger = logger;
+            _baseStore = baseStore;
+        }
+
+        public override Task<GetResponse> Get(GetRequest request, ServerCallContext context)
+        {
+            var key = new KeyBytes(request.Key.ToByteArray());
+            var result = _baseStore.TryGet(key, out var value);
+            _logger.LogTrace("Get: {Key} => {Value}", key, value);
+            return Task.FromResult(new GetResponse { Value = ByteString.CopyFrom(value), Exists = result});
+        }
+
+        public override Task<SetResponse> Set(SetRequest request, ServerCallContext context)
+        {
+            var key = new KeyBytes(request.Key.ToByteArray());
+            _baseStore.Set(key, request.Value.ToByteArray());
+            _logger.LogTrace("Set: {Key} <= {Value}", key, request.Value);
+            return Task.FromResult(new SetResponse());
+        }
+
+        public override Task<SetMapResponse> SetMap(SetMapRequest request, ServerCallContext context)
```

Comment:
I was bit confused by the name. Why not `SetCollection` like `DeleteCollection`?

Reply:
Multiple `Set` gets `Dictionary`. On the other side, multiple `Delete` gets `List`.

Reply:
https://github.com/planetarium/libplanet/pull/3688/files#diff-baf92b08d85437de7131f62dd9b70066064c72f21b241ca587248ef293c5833bR17 fyi.

Reply:
I understand. However, the term "collection" is underlying data type agnostic. I'd suggest either

```proto
message SetCollectionRequest {
    map<string, bytes> key = 1;
}

message DeleteCollectionRequest {
    repeated bytes key = 1;
}
```

or 

```proto
message SetMapRequest {
    map<string, bytes> map_field = 1;
}

message DeleteRepeatedRequest {
    repeated bytes repeated_field = 1;
}
```

for consistency. I prefer the former, but maybe I don't know enough about Protobuf's general conventions. 🙄

Reply:
https://cloud.google.com/apis/design/naming_convention?hl=ko#method_names

Ok, I found it. It said: `Use *VerbNone* naming convention`. So, I suggest...

```
message SetPairsRequest {
    map<string, bytes> map_field = 1;
}

message DeleteListRequest {
    repeated bytes repeated_field = 1;
}
```

How about this? @greymistcube

Reply:
Looks good to me. 🙂

Reply:
https://github.com/planetarium/libplanet/pull/3688/commits/60fb78868f5bf781e20ef631281fb048c2669955 amended. PTAL 🙏 @greymistcube